doc/source/_templates/sidebar-nav-bs.html

```diff
@@ -1,9 +1,9 @@
 <nav class="bd-links" id="bd-docs-nav" aria-label="Main navigation">
   <div class="bd-toc-item active">
     {% if pagename.startswith("reference") %}
-    {{ generate_nav_html("sidebar", maxdepth=4, collapse=True, includehidden=True, titles_only=True) }}
+    {{ generate_nav_html("sidebar", maxdepth=2, collapse=True, includehidden=True, titles_only=True) }}
```

Comment:
```suggestion
    {{ generate_nav_html("sidebar", maxdepth=1, collapse=True, includehidden=True, titles_only=True) }}
```

My previous comment was mainly about the user guide, I think you *could* keep it here at a maxdepth of 1 in the reference guide, because it's mainly here that you get such huge sidebars because of all individual docstring pages getting included.